clients/cobol-lsp-vscode-extension/src/__tests__/services/LanguageClientServiceTest.spec.ts

```diff
@@ -42,6 +42,7 @@ jest.mock("vscode", () => ({
       fsPath: "path",
     }),
   },
+  RelativePattern: jest.fn().mockReturnValue(undefined),
```

Comment:
Why is this needed?

Reply:
We need to mock it because extension activate method triggers it via : https://github.com/hizir-bc/che-che4z-lsp-for-cobol/blob/feature/swap-workspace-home/clients/cobol-lsp-vscode-extension/src/extension.ts#L58

Reply:
I see, I didn't realize that whole vscode is being mocked here.